plugins/file-open-save-new/core/src/main/javascript/app/app.component.js

```diff
@@ -458,38 +457,6 @@ define([
       }
     }
 
-    /**
```

Comment:
code is no longer needed, from backlog-31207 the function vm.onOpenClick() replaced functionality.